lib/ex_gram/updates/polling.ex

```diff
@@ -41,7 +44,9 @@ defmodule ExGram.Updates.Polling do
   end
 
   @default_opts [limit: 100, timeout: 50]
-  defp get_updates(token, uid, opts \\ []) do
+  defp get_updates(token, uid, opts) do
+    opts = Keyword.take(opts, [:allowed_updates])
```

Comment:
```suggestion
  defp get_updates(token, uid, opts) do
    opts = Keyword.take(opts, [:allowed_updates, :limit, :timeout])
```

We still want to be able to configure limit and timeout no?